core/whistle-1.0.0/src/wh_util.erl

```diff
@@ -21,8 +21,13 @@
          ,is_system_db/1
         ]).
 -export([get_account_realm/1, get_account_realm/2]).
--export([is_account_enabled/1]).
--export([is_account_expired/1]).
+-export([is_account_enabled/1, is_account_expired/1]).
+-export([maybe_disable_account/1
```

Comment:
I think we should move save operations there, especially for account docs, since they need to be persisted to the account db and the `accounts` db. Good to have it centrally located.

I would add a save/1 to kz_account, and add an optional third param to the set functions of whether to save the set (defaulting to false). It should return the updated account document (including new _rev) or return the error tuple.

Something like:

```erlang
set_superduper_admin(AccountJObj, IsAdmin) ->
    set_superduper_admin(AccountJObj, IsAdmin, 'false').
set_superduper_admin(AccountJObj, IsAdmin, ShouldSave) ->
    Updated = wh_json:set_value(...),
    maybe_persist(Updated, ShouldSave).

maybe_persist(JObj, 'false') -> JObj;
maybe_persist(JObj, 'true') ->
    case save(JObj) of
        {'ok', Saved} -> Saved;
        {'error', _R}=E -> E
    end.
```

Seems like a good pattern to follow for the setters.